### ISSUE TITLE
Use Ansible vault encryption instead of python-keyczar

### ISSUE THREAD

**Issue by racke** [sympa-community/sympa-ansible#33]
It is certainly possible to use standard Ansible vault for encryption of single variables in a YAML file despite the claim on https://github.com/sympa-community/sympa-ansible#encrypting-secrets. So we could make the encryption of these variables the **default**, especially as we generate the environment.

**Comment by dverdin** [comment]
A note: It is the default now for all passwords and certficates generated.

**Comment by racke** [comment]
As far as I can tell, Ansible Vault does *not* use python-keyczar. Thus this issue.

**Comment by dverdin** [comment]
Oh. I thought you wanted the passwords to be encrypted by default. No: you want them encrypted by default by Ansible.
OK.

**Comment by racke** [comment]
I don't see the benefit of python-keyczar over the native Ansible vault. Also I doesn't seem to be a good decision security wise (https://blog.mkowalski.net/2017/02/13/the-uncertain-future-of-keyczar/, https://github.com/google/keyczar).

**Comment by dverdin** [comment]
You know, we are using keyczar because we have a working solution based on it.
If there is another mean to:
  * encrypt only the secrets,
  * have a key for decrypting/encrypting them
As far as I'm concerned, I don't care about the solution employed.
Feel free to PR once you found something! :-)

**Comment by racke** [comment]
Debian is going to remove it, so I don't think it is viable solution anymore:

```
python-keyczar (0.716+ds-3) unstable; urgency=medium

   This package will be removed from the archive soon. The project is no longer
   maintained upstream, and it contains known security issues, see:

       https://github.com/google/keyczar#known-security-issues

 -- Christian Kastner <ckk@debian.org>  Fri, 15 Feb 2019 21:00:21 +0100
```

**Comment by dverdin** [comment]
That indeed put another degree of urgency to the question.
We'll dig in the ansible solution as soon as possible.

**Comment by racke** [comment]
Actually the security problems were known when you started to use it for refactoring. Any way, let's find a better solution.

**Comment by racke** [comment]
Will be fixed by #70

**Comment by racke** [comment]
Finally done.
